### PR TITLE
Spin off package.json scripts into a new directory

### PR DESCRIPTION
**Pull request by gnilekaw**
This PR breaks out the package.json `scripts` field into separate files. I'd like to merge this initial pass before moving on to [Circle](https://github.com/artsy/force/blob/master/circle.yml#L43). Thanks to @alloy for suggesting using `set` instead of `&& \`s all over the place.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by craigspaeth** [comment]
LGTM, maybe our [Circle](https://github.com/artsy/force/blob/master/circle.yml#L23) scripts can be moved/consolidated here too.

**Comment by gnilekaw** [comment]
@craigspaeth I'm hoping to tackle that next! I just didn't want to start to ambitiously, haha.

**Comment by craigspaeth** [comment]
Sounds good! Feel free to merge when you're ready :)

**Comment by gnilekaw** [comment]
Sorry this has been sitting around so long! Hoping to get back to it this week. Otherwise I'll close it and reopen when I have a chance.

File: scripts/deploy-assets.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# !/usr/bin/bash
+
+NODE_ENV=production mkdir public; \
+mkdir public/assets; \
+ezel-assets mobile/assets/ && \
+ezel-assets desktop/assets/ && \
+bucket-assets --bucket artsy-force-$DEPLOY_ENV && \
```
Comment (alloy) [inline scripts/deploy-assets.sh]: You can also start  your script with `set -e` so you don’t need to add all the `&& \` parts.

```
sh-3.2$ help set
set: set [--abefhkmnptuvxBCHP] [-o option] [arg ...]
        -e  Exit immediately if a command exits with a non-zero status.
        -x  Print commands and their arguments as they are executed.
```

May also want to set that second one.

File: scripts/test.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+yarn mocha $(find desktop/test -name '*.coffee') && \
+yarn mocha $(find desktop/components/*/test -name '*.coffee') && \
+yarn mocha $(find desktop/components/**/*/test -name '*.coffee') && \
+yarn mocha $(find desktop/apps/*/test -name '*.coffee' -or -name '*.js') && \
+yarn mocha $(find desktop/apps/*/**/*/test -name '*.coffee') && \
+yarn mocha $(find mobile/test -name '*.coffee') && \
+yarn mocha $(find mobile/components/*/test -name '*.coffee') && \
+yarn mocha $(find mobile/components/**/*/test -name '*.coffee') && \
+yarn mocha $(find mobile/apps/*/test -name '*.coffee') && \
+yarn mocha $(find mobile/apps/*/**/*/test -name '*.coffee')
```
Comment (alloy) [inline scripts/test.sh]: Ditto

Comment (alloy) [inline scripts/test.sh]: These `&& \` are no longer necessary

Comment (gnilekaw) [inline scripts/test.sh]: Ack!

**Comment by craigspaeth** [comment]
LGTM, thanks!

**Pull request merged**